src/foam/nanos/auth/Relationships.js

```diff
@@ -48,8 +48,8 @@ foam.RELATIONSHIP({
   cardinality: '1:*',
   sourceModel: 'foam.nanos.auth.UserUserJunction',
   targetModel: 'foam.nanos.notification.NotificationSetting',
-  forwardName: 'notificationSettingsForBusinessUsers',
-  inverseName: 'businessUser',
+  forwardName: 'notificationSettingsForUserUsers',
```

Comment:
Does this break calls to `businessUserJunction.getNotificationSettingsForBusinessUsers(x)`?

Reply:
Yes it will.

Reply:
I renamed this because references to businesses shouldn't exist in foam unless it's explicitly required.